docs/pages/versions/unversioned/sdk/sqlite-next.mdx

```diff
@@ -166,23 +164,61 @@ const styles = StyleSheet.create({
 });
 ```
 
+### `useSQLiteContext()` hook with `React.Suspense`
+
+As with the [`useSQLiteContext()` hook](#usesqlitecontext-hook), you can also integrate the `SQLiteProvider` with [`React.Suspense`](https://react.dev/reference/react/Suspense) to show a fallback component until the database is ready.
+To enable the integration, you need to pass the `useSuspense` prop to the `SQLiteProvider` component.
```

Comment:
```suggestion
As with the [`useSQLiteContext()`](#usesqlitecontext-hook) hook, you can also integrate the `SQLiteProvider` with [`React.Suspense`](https://react.dev/reference/react/Suspense) to show a fallback component until the database is ready. To enable the integration, pass the `useSuspense` prop to the `SQLiteProvider` component.
```

- For internal links, let's only highlight the actual hook/method name
- We can be a bit concise in the last sentence
- Since there is no new line break, i don't think we have to divide this paragraph into to new lines.